src/components/note/note-test.stories.tsx

```diff
@@ -38,3 +41,60 @@ DefaultStory.story = {
     createdDate: "23 May 2020, 12:08 PM",
   },
 };
+
+export const InlineControlMenuButton = ({
+  name,
+  createdDate,
+  ...args
+}: Omit<NoteProps, "noteContent">) => {
+  const html = `<p>Lorem ipsum <b>dolor</b> sit amet, <i>consectetuer adipiscing elit.</i> Aenean commodo ligula eget dolor. <b><i>Aenean massa.</i></b></p>
+  <ul><li>unordered</li></ul>
+  <ol><li>ordered</li></ol></br>
+  <p>Lorem ipsum <b>dolor</b> sit amet, <i>consectetuer adipiscing elit.</i></p>
+  <p>Aenean commodo ligula eget dolor. <b><i>Aenean massa.</i></b></p>`;
+  const blocksFromHTML = convertFromHTML(html);
+  const content = ContentState.createFromBlockArray(
+    blocksFromHTML.contentBlocks,
+    blocksFromHTML.entityMap
+  );
+  const noteContent = EditorState.createWithContent(content);
+
+  const inlineControl = (
+    <ActionPopover
+      renderButton={(props) => (
+        <ActionPopoverMenuButton
+          buttonType="tertiary"
+          iconType="ellipsis_vertical"
+          iconPosition="after"
+          size="small"
+          {...props}
+        >
+          Actions
+        </ActionPopoverMenuButton>
+      )}
+    />
+  );
+
+  return (
+    <div style={{ height: 300, width: "50%" }}>
+      <Note
+        title="Here is a Title"
+        inlineControl={inlineControl}
+        noteContent={noteContent}
+        name={name}
+        createdDate={createdDate}
+        {...args}
+      />
+    </div>
+  );
+};
+
+InlineControlMenuButton.story = {
+  name: "inline control menu button",
+  args: {
+    name: "Lauren Smith",
+    createdDate: "23 May 2020, 12:08 PM",
+  },
+};
+
+InlineControlMenuButton.parameters = { chromatic: { disableSnapshot: false } };
```

Comment:
suggestion: as there's no real theme differences we can probably set the it to `sage` rather than create a snapshot for each one 

```
themeProvider: { chromatic: { theme: "sage" } },

Reply:
Sorry I think I got this slightly wrong it should be:

```
themeProvider: { chromatic: { disableSnapshot: false, theme: "sage" } },